app/components/Vault/index.js

```diff
@@ -605,10 +610,15 @@ const Vault = (props) => {
   if (vault.fees && vault.fees.general) {
     if (v2Vault) {
       const { managementFee, performanceFee } = vault.fees.general;
-      const keepCrv =
-        vault.fees && vault.fees.special.keepCrv > 0
-          ? vault.fees.special.keepCrv
+      let keepCrv =
+        vault.apy &&
+        vault.apy.fees &&
+        vault.apy.fees.keep_crv &&
+        vault.apy.fees.keep_crv >= 0
```

Comment:
could probably replace with `_.get(vault, 'apy.fees.keep_crv')` but this is also find and i understand not wanting to rely on lodash